lib/CMakeLists.txt

```diff
@@ -78,15 +78,15 @@ if(ENABLE_SHARED_LIB)
     RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
 endif()
 
-# Static library (for unittests because of symbol visibility)
-add_library(nghttp3_static STATIC ${nghttp3_SOURCES})
-set_target_properties(nghttp3_static PROPERTIES
-  COMPILE_FLAGS "${WARNCFLAGS}"
-  VERSION ${LT_VERSION} SOVERSION ${LT_SOVERSION}
-  ARCHIVE_OUTPUT_NAME nghttp3${STATIC_LIB_SUFFIX}
-  )
-target_compile_definitions(nghttp3_static PUBLIC "-DNGHTTP3_STATICLIB")
 if(ENABLE_STATIC_LIB)
```

Comment:
Perhaps an empty line here to keep the layout in-sync with https://github.com/ngtcp2/ngtcp2/pull/1189/files?w=1#diff-4803677bd280f4dded52c428e3c8ac6c618508dcabcbe880b68be85897c2ad41L104